scripts/ci/build_emscripten.sh

```diff
@@ -56,8 +56,7 @@ fi
 
 mkdir -p $BUILD_DIR
 cd $BUILD_DIR
-cmake \
-  -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/emscripten.cmake \
+emcmake cmake \
```

Comment:
This is the canonical way to build cmake projects with emscripten. No idea why we actually ever bothered with the toolchain file mess...